src/main/java/org/cascadebot/cascadebot/music/KaraokeHandler.java

```diff
@@ -0,0 +1,112 @@
+package org.cascadebot.cascadebot.music;
+
+import net.dv8tion.jda.api.EmbedBuilder;
```

Comment:
In my opinion, the Cascade imports and the JDA imports should be first, then everything else

Reply:
We use Intellijs default import sorting which is what this is